gradle.properties

```diff
@@ -15,7 +15,7 @@ mockitoVersion=1.10.19
 
 # Version and base tags can be overridden at build time.
 connectorVersion=0.2.0-SNAPSHOT
-pravegaVersion=0.2.0-SNAPSHOT
+pravegaVersion=0.3.0-SNAPSHOT
```

Comment:
Pravega `master` is now in 0.3.0-SNAPSHOT. Not sure if we need to port the changes to 0.2.0-SNAPSHOT branch as well?

Reply:
I think you're referring to the connector's `release-nautilus-0.3` branch.   These changes are not needed there.

Reply:
No. I am referring to current `master` from which we may have to cut a release branch `release-nautilus-0.4` for Pravega 0.2.0 release support.